operations/mimir-mixin/config.libsonnet

```
@@ -96,5 +96,10 @@
     // The default datasource used for dashboards.
     dashboard_datasource: 'default',
     datasource_regex: '',
+
+    // Tunes histogram recording rules to aggregate over this interval.
+    // Set to at least twice the scrape interval; otherwise, recording rules will output no data.
+    // Set to four times the scrape interval to account for edge cases: https://www.robustperception.io/what-range-should-i-use-with-rate/
+    rules_range: '1m',
```

WDYT about make it a bit more verbose for clarity?

```suggestion
    recording_rules_range_interval: '1m',
```